menpo/io/output/base.py

```diff
@@ -113,7 +113,22 @@ def export_pickle(obj, fp, overwrite=False):
         The string path or file-like object to save the object at/into.
     overwrite : `bool`, optional
         Whether or not to overwrite a file if it already exists.
-
+    protocol : `int`, optional
+        Save with a different pickle protocol the file.
```

Comment:
Change this line to just 'The Pickle protocol used to serialize the file'. The 'change default' but is redundant - that's implied by the fact it's an optional kwarg. Picky I know! 🙊